src/main/java/com/salesforce/dataloader/ui/Grid12.java

```diff
@@ -165,4 +172,11 @@ public void show(Control control) {
     public void pack() {
         composite.pack();
     }
+
+    private void applyFormatting(Control control) {
+        //use this to apply standard formatting
+
```

Comment:
yeah. I put all this code here for debugging. We don't need any standard formatting but I don't want to create it again next time I need to see the grids on the UI. I can pull it if you think it's better.